public/app/features/scopes/internal/ScopesSelectorScene.tsx

```diff
@@ -358,6 +368,7 @@ export function ScopesSelectorSceneRenderer({ model }: SceneComponentProps<Scope
               onClick={() => {
                 model.closePicker();
                 model.updateScopes();
+                model.toggleDashboardsPanel();
```

Comment:
I would rather make this be called from `updateScopes` in order to be called for all sorts of scopes calls (i.e. updating scopes from the URL).

If we do this, we can also pass the "computed" scopes from there so we get the up-to-date list of scopes.